DEV_GUIDE.md

```diff
@@ -77,7 +77,7 @@ docker pull stargateio/stargate-3_11:v0.0.2
 ```
 
 ```sh
-docker run --name stargate -d stargateio/stargate-3_11:v0.0.2 --cluster-name stargate --cluster-seed 127.0.0.1 --cluster-version 3.11 --listen 127.0.0.2 --simple-snitch
+docker run --name stargate -d stargateio/stargate-3_11:v0.0.2 --cluster-name stargate --cluster-seed 127.0.0.1 --cluster-version 3.11 --listen 127.0.0.2 --bind-to-listen-address --simple-snitch
```

Comment:
Were you getting a port collision on 9042 without this?

Reply:
Yes. Without bind-to-listen-address CQL native transport defaults to 0.0.0.0:9042 which conflicts with 127.0.0.1:9042 (used by C* started with ccm).

Reply:
Could be caused by the old version of the container, perhaps? Does the latest container also get port conflicts?

Reply:
@dimas-b I've just realized that I made a mistake. The flag was supposed to be added to starctl cmd, not to `docker run`. Sorry for the confusion. Here is a PR that moves the flag to the right place: #705.